spec/latest/core/index.html

```diff
@@ -631,6 +631,23 @@ <h3>Discovering a Hydra-powered Web API</h3>
       used in responses from the API will dereference to their
       documentation.</p>
   </section>
+
+  <section>
+    <h3>Api versions</h3>
+
+    <p>It is common to provide a separate Api address after a breaking changes
```

Comment:
I think `API` is capitalised throughout the spec. I'd stick to single spelling

```suggestion
    <p>It is common to provide a separate API address after a breaking changes
```